src/__tests__/App.test.js

```diff
@@ -0,0 +1,87 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import { Provider } from "react-redux";
+import history from "../history";
+import createStore from "../store/store";
+import axios from "../axios";
+import App from "../App";
+import { AUTHORIZATION } from "../store/actions";
+
+jest.mock("../axios");
+
+let store;
+beforeEach(() => {
+  localStorage.clear();
+  store = createStore();
+});
+
+test("renders login component", async () => {
+  history.push("/login");
+  render(
+    <Provider store={store}>
+      <App />
+    </Provider>
+  );
+  await waitFor(() =>
+    expect(screen.getAllByText(/Login/i)[0]).toBeInTheDocument()
+  );
+});
+
+test("renders PetListPage component", async () => {
+  axios.get.mockResolvedValueOnce({
```

Comment:
Please try to check these three axios.get.mockResolvedValueOnce, maybe the result is returning last axios.get

Reply:
Hi!
Thank you very much for your feedback, there are 3 axios mocks because:
1. The first one mocks the user info that is logged in.
2. The second one mocks all the pets list that comes from MongoDB.
3. The third one mocks all the adoption requests from a specific foundation

Based on this, all the requests mocks get different info, hence they are required.

Thank you!